### ISSUE TITLE
withDevices: in scan method

### ISSUE THREAD

**Issue by ericstephen** [pauldemarco/flutter_blue#286]
If not a device uuid what is the purpose of this parameter? No matter what I put here the scan method will always scan for all devices. What am I misunderstanding?

**Comment by bpillon** [comment]
Hello Eric.
For the moment, withDevices seems to be useless. You can look at the scan method implementation, the parameter withDevices is never user;

**Comment by bpillon** [comment]
After a fast look at native documentations. It seems that iOS doesn't allow to scan only some devices (it's possible on Android). But we could may be implement a filter on the result on iOS to send to Flutter only the post filtered list.

**Comment by ericstephen** [comment]
Yes, and I have been working on the iOS platform with no testing on Android at the moment. It would be nice to use this on iOS to filter out the noise of other bluetooth devices in the scan and I know I can add a service to the advertisement data, but I am unable to change that at the moment so was hoping for an alternative means to filter out the noise.
